src/Storefront/Test/Framework/Cache/CacheKeyProviderTest.php

```diff
@@ -0,0 +1,74 @@
+<?php declare(strict_types=1);
+
+namespace Shopware\Storefront\Test\Framework\Cache;
+
+use PHPUnit\Framework\TestCase;
+use Shopware\Core\Framework\Test\TestCaseBase\IntegrationTestBehaviour;
+use Shopware\Storefront\Framework\Cache\CacheKeyProvider;
+use Symfony\Component\HttpFoundation\Request;
+
+class CacheKeyProviderTest extends TestCase
+{
+    use IntegrationTestBehaviour;
```

Comment:
Use KernelTestBehaviour trait. IntegrationTestBehaviour has lot of cleanup , db transaction stuff which is not needed